.s2i/httpd-cfg/01-commercial.conf

```diff
@@ -175,15 +175,50 @@ AddType text/vtt                            vtt
     RewriteRule ^container-platform/4\.10/networking/external_dns_operator/nw-installing-external-dns-operator.html /container-platform/4.10/networking/external_dns_operator/nw-installing-external-dns-operator-on-cloud-providers.html [NE,R=301]
 
-   # Redirect for renamed Ingress Controller document
+    # Redirect for renamed Ingress Controller document
     RewriteRule ^container-platform/(4\.1[2-6])/networking/nw-ingress-controller-endpoint-publishing-strategies\.html$ /container-platform/$1/networking/nw-configuring-ingress-controller-endpoint-publishing-strategy.html [NE,R=302,L]
 
     # Redirect for Networking operators move per https://github.com/openshift/openshift-docs/pull/85085
     RewriteRule ^container-platform/(4\.1[6-8])/networking/aws_load_balancer_operator/(.*)? /container-platform/$1/networking/networking_operators/aws_load_balancer_operator/$2 [NE,R=302]
+
+    RewriteRule rosa/networking/aws-load-balancer-operator.html /container-platform/$1/networking/networking_operators/aws-load-balancer-operator.html [NE,R=302]
+
     RewriteRule ^container-platform/(4\.1[6-8])/networking/ingress-operator.html /container-platform/$1/networking/networking_operators/ingress-operator.html [NE,R=302]
+
     RewriteRule ^container-platform/(4\.17|4\.18)/networking/network_security/ebpf_manager/(.*)? /container-platform/$1/networking/networking_operators/ebpf_manager/$2 [NE,R=302]
+    
     RewriteRule ^container-platform/(4\.17|4\.18)/networking/external_dns_operator/(.*)? /container-platform/$1/networking/networking_operators/external_dns_operator/$2 [NE,R=302]
 
+    # PR https://github.com/openshift/openshift-docs/pull/85085
+    # Redirect for the About MetalLB and the MetalLB Operator document
+    RewriteRule ^container-platform/(4\.1[6-8])/networking/metallb/about-metallb.html /container-platform/$1/networking/networking_operators/metallb-operator/about-metallb.html [NE,R=302]
+    # Redirect for the Installing the MetalLB Operator document
+    RewriteRule ^container-platform/(4\.1[6-8])/networking/metallb/metallb-operator-install.html /container-platform/$1/networking/networking_operators/metallb-operator/metallb-operator-install.html [NE,R=302]
+    # Redirect for the Upgrading the MetalLB Operator document
+    RewriteRule ^container-platform/(4\.1[6-8])/networking/metallb/metallb-upgrading-operator.html /container-platform/$1/networking/networking_operators/metallb-operator/metallb-upgrading-operator.html [NE,R=302]
+    # Redirect for the Cluster Network Operator document
+    RewriteRule ^container-platform/(4\.1[6-8])/networking/cluster-network-operator.html /container-platform/$1/networking/networking_operators/cluster-network-operator.html [NE,R=302]
+    # Redirect for the DNS Operator document
+    RewriteRule ^container-platform/(4\.1[6-8])/networking/dns-operator.html /container-platform/$1/networking/networking_operators/dns-operator.html [NE,R=302]
+    # ROSA/Dedicated Redirect for the DNS Operator document
```

Comment:
Hi @bscott-rh . Is this syntax correct? Rosa and Dedicated redirects are also needed :grimacing: